<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalMattingWidget</class>
 <widget class="QWidget" name="GlobalMattingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>611</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>933</width>
    <height>611</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPixmapLabel" name="inputLabel">
     <property name="minimumSize">
      <size>
       <width>455</width>
       <height>294</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>520</width>
       <height>336</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPixmapLabel" name="trimapLabel">
     <property name="minimumSize">
      <size>
       <width>454</width>
       <height>294</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>520</width>
       <height>335</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPixmapLabel" name="outputMaskLabel">
     <property name="minimumSize">
      <size>
       <width>454</width>
       <height>293</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPixmapLabel" name="outputColorLabel">
     <property name="minimumSize">
      <size>
       <width>455</width>
       <height>293</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>520</width>
       <height>335</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPixmapLabel</class>
   <extends>QLabel</extends>
   <header>QPixmapLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
